docs/dxp/7.x/en/site-building/site-appearance/site-appearance.md

```diff
@@ -12,46 +12,33 @@ The design and functionality of a Liferay site can be adjusted and customized th
 
 ## Style Books
 
-[Screenshot of creating a style book?]
-
-Introduce Style Books, note a few really general use cases to give the idea. Note that all the levels "above" (page templates, fragments, etc.) all use these in their displays
-
-## Widget/Fragment Appearance
-
-[Maybe show a considerably customized-looking widget/fragment?]
+[Style Books](./style-books/using-a-style-book-to-standardize-site-appearance.md) are sets of visual standards that apply to a Site, such as standard colors and spacing. This allows you to design every page's layout independently while maintaining a consistent experience between each of them.
 
-Introductions and links to:
-* Fragment customizations
-* Collection Display fragment renderers? Any other analogous features for other fragments? (Hopefully not too long of a list)
-* Application Display Templates
-* Maybe also a reference to the Configuring Fragment Style article
+![Style Books define the rules for your Site's appearance, such as text colors.](./site-appearance/images/02.png)
 
-Note that these can be embedded into many of the lower "layers"
-* Does that mean order should be reversed? Feels more natural to start with lowest level though, esp. since that puts themes last
+<!--
+TODO:
 
 ### Style Book Token Definitions
 
 Note how they tie into themes as well (depending on them for definitions), possibly? And then maybe link to developer guide articles? (Unless maybe developer guide material is not good to go into for this overview... then perhaps it'd be better to just briefly reference their reliance on the theme and leave it as that, not even with an H3)
+-->
 
-## Themes
+## Widget/Fragment Appearance
 
-[Screenshot of page with non-Classic theme?]
+Fragments and widgets are the building blocks of your Site. There are many ways to customize them to suit your Site's appearance.
 
-Introduce themes, give picture of how they can be highly flexible.
+* [Fragment customizations](../developer-guide/developing-page-fragments/developing-fragments-intro.md) can be used to tailor the look and feel of your Site's Fragment Pages
+* Application Display Templates<!--Add link when available--> allow you to customize the appearance and behavior of the widgets on your Site
 
-Reference how they tie into all the higher "layers"?
+## Themes
 
-* Fragments/widgets use themes, and can also be embedded within the themes if necessary
-* Page templates use themes for styling
-* Style books rely on themes for their token definitions
+[Themes](./themes/introduction-to-themes.md) are a highly flexible tool for customizing the default look and feel of your Site. They provide a foundation that your [Style Books](#style-books) and [Page Templates](../creating-pages/adding-pages/creating-a-page-template.md) build on top of. Fragments and themes can also be embedded in the themes themselves to ensure a particular arrangement.
```

Comment:
I'm not sure about this fragment: "Fragments and themes can also be embedded in the themes themselves to ensure a particular arrangement"

I think that it "Fragment" shall be changed by "Widget". Regarding "themes", I'm not sure what you mean. Are you referring to theme inheritance? Maybe "templates" instead?

Reply:
I think that was a mix-up on my part, and I meant to say "fragments and widgets can also be embedded in themes." (It is true that fragments can be embedded in the themes, too, right?)

Reply:
I am not 100% sure about being able to add fragments to the theme. I've found a taglib to render a fragment, which _theoretically_, shall be usable from a template in a theme very much like the runtime tag we use for widgets. I'd still refrain from mentioning this as I saw no usages of it yet.
But maybe in the future we'll have to revisit this if we see interest in this feature